project/cfg_utils.py

```diff
@@ -0,0 +1,10 @@
+from pyformlang.cfg import CFG
+
+
+def cfg_to_normal_form(cfg: CFG) -> CFG:
+    return cfg.to_normal_form()
```

Comment:
It seems, that the result grammar is not equal to the source in some cases, note about weak cnf